lxd/api_project.go

```diff
@@ -168,7 +167,7 @@ func projectCreateDefaultProfile(tx *db.ClusterTx, project string) error {
 	profile.Name = "default"
 	profile.Description = fmt.Sprintf("Default LXD profile for project %s", project)
 	profile.Config = map[string]string{}
-	profile.Devices = config.Devices{}
+	profile.Devices = map[string]map[string]string{}
```

Comment:
Do we even need to set profile.Config and profile.Devices at all? Could you try just removing those two lines and see if things blow up in tests (I don't see why it would)?

Reply:
@stgraber I will try it, map's zero value is nil, so it may blow out but worth a try.

Reply:
Yeah, but we support someone not passing it through the REST API (or we should anyway), so it shouldn't blow up.

Reply:
@stgraber made those changes, seems fine.